modules/nw-externalip-about.adoc

```diff
@@ -0,0 +1,189 @@
+====
+
+// Do we offer this in 4.x?
+// Also, these addresses can be used as virtual IPs (VIPs) when configuring xref../admin_guide/high_availability.adoc#configuring-ip-failover[high availability].
+
+{product-title} supports both the automatic and manual assignment of IP
+addresses, and each address is guaranteed to be assigned to a maximum of one
+service. This ensures that each service can expose its chosen ports regardless
+of the ports exposed by other services.
+
+For manually-configured external IPs, potential port clashes are handled on a first-come, first-served basis. If you request a port, it is only available if it has not yet been assigned for that IP address. For example:
+
+.Port collision example for manually-configured external IP addresses
+In the following example, two services have been manually configured with the same external IP address of 172.7.7.7.
+
+`MongoDB service A` requests port 27017, and then
+`MongoDB service B` requests the same port; the first request gets the port.
+
+However, port collisions are not an issue for external IP addresses assigned by the ingress controller, because the controller assigns each service a unique address.
+
+[NOTE]
+====
+To use IP address blocks defined by `autoAssignCIDRs` in {product-title}, you must configure the necessary IP address assignment and routing for your host network.
+
+A solution such as ipfailover is recommended to ensure high availability of the IP address pools.
+
+Both have to be configured to a device on an {product-title} host to be used,
+whether with network interface controller (NIC) or virtual ethernet, as well as
+external routing. Ipfailover is recommended for this, because it selects the
+host and configures the NIC.
+====
+
+The following YAML describes a Service spec with an external IP configured:
+
+.Example Service object with `spec.externalIPs[]` set
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: http-service
+spec:
+  clusterIP: 172.30.163.110
+  externalIPs:
+  - 192.168.132.253
+  externalTrafficPolicy: Cluster
+  ports:
+  - name: highport
+    nodePort: 31903
+    port: 30102
+    protocol: TCP
+    targetPort: 30102
+  selector:
+    app: web
+  sessionAffinity: None
+  type: LoadBalancer
+status:
+  loadBalancer:
+    ingress:
+    - ip: 192.168.132.253
+----
+
+[id="automatic-external-ip_assignment_{context}"]
+== Automatic assignment of an external IP
+
+{product-title} automatically assigns an IP address from the `autoAssignCIDRs` CIDR range to the `spec.externalIPs[]` array when you create a Service CR object with `spec.type=LoadBalancer`.
+In this case, {product-title} implements a non-cloud version of the load balancer service type and assigns IP addresses to the services.
+
+[id="manual-external-ip_assignment_{context}"]
+== Manual assignment of an external IP
+
+{product-title} uses the IP addresses assigned to the `spec.externalIPs[]` array when you create a Service CR object with `spec.type=LoadBalancer`.
```

Comment:
I do not think Manual assignment of an external IP need a Service CR object with `spec.type=LoadBalancer`.